build/run_in_workspace_with_goroot.bzl

```diff
@@ -51,18 +51,15 @@ _workspace_binary_script = rule(
     attrs = {
         "cmd": attr.label(
             mandatory = True,
-            allow_files = True,
-            single_file = True,
+            allow_single_file = True,
```

Comment:
This was needed for compatibility with Bazel 0.27, and is supported all the way back to our minimum required version of 0.23